packages/picasso/src/NativeSelectOptions/NativeSelectOptions.tsx

```diff
@@ -12,20 +13,36 @@ const NativeSelectOptions = <T extends ValueType>({
   options,
   renderOption,
   getItemProps
-}: Props<T>) => (
-  <>
-    {options.map((option, index) => (
-      <option
-        key={(option?.key ?? option.value).toString()}
-        value={option.value.toString()}
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...getItemProps(option, index)}
-      >
-        {renderOption(option, index)}
-      </option>
-    ))}
-  </>
-)
+}: Props<T>) => {
+  const renderOptions = (optionsList: Option<T>[]) => (
```

Comment:
We understand that it's a list of options from typing. Let's not duplicate it in the variable name.

```suggestion
  const renderOptions = (options: Option<T>[]) => (
```

Reply:
`options` variable is already used in the upper scope. I had to use a different name for this argument.

Reply:
You could use `flatOptions`.